tests/phpunit/tests/functions/wpuploadbits.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Test the wp_upload_bits function
+ *
+ * @group Functions
+ * @group Upload
+ * @covers ::wp_upload_bits
+ */
+
+class tests_wp_upload_bits extends WP_UnitTestCase {
+
+	/**
+	 * @expectedDeprecated
```

Comment:
```suggestion
	 * @expectedDeprecated wp_upload_bits
```
The `@expectedDeprecated` annotation must be followed by the function you expect to throw a deprecation notice.